types/wordpress__components/combobox-control/index.d.ts

```diff
@@ -0,0 +1,19 @@
+import { ComponentType, ReactNode } from 'react';
+
+declare namespace ComboboxControl {
+    interface Props {
+        value: string | number;
+        label?: ReactNode | undefined;
```

Comment:
```suggestion
        label?: string | undefined;
```

I think `label` has to be a `string` because we do a string concat with it.
https://github.com/WordPress/gutenberg/blob/0135921bcb72ceec629fa8f9e755891afab747b9/packages/components/src/combobox-control/index.js#L235

Reply:
Ah, thanks, I didn't see that. I just followed that it was passed along to `label` on `BaseControl`, which accepts a `ReactNode`.